src/fides/api/service/connectors/sql_connector.py

```diff
@@ -176,6 +185,24 @@ def set_schema(self, connection: Connection) -> None:
         """Optionally override to set the schema for a given database that
         persists through the entire session"""
 
+    def create_ssh_tunnel(self, host: Optional[str], port: Optional[int]) -> None:
+        """Creates an SSH Tunnel to forward ports as configured."""
+        if CONFIG.security.bastion_server_ssh_pkey:
```

Comment:
Should we raise a custom exception here if this is not set, but `ssh_required` has led Fides down this path?

Reply:
That seems like a great idea, thanks!